charts/hedera-json-rpc-relay-websocket/templates/configmap.yaml

```diff
@@ -6,22 +6,4 @@ metadata:
     app:  {{ template "json-rpc-relay-ws.name" . }}
     {{ include "json-rpc-relay-ws.labels" . | nindent 4 }}
 data:
-  CHAIN_ID: {{ .Values.config.CHAIN_ID | quote }}
-  DEV_MODE: {{ .Values.config.DEV_MODE | quote }}
-  HEDERA_NETWORK: {{ required "A valid HEDERA_NETWORK must be present in" .Values.config.HEDERA_NETWORK | toJson | quote }}
-  MIRROR_NODE_LIMIT_PARAM: {{ .Values.config.MIRROR_NODE_LIMIT_PARAM | quote }}
-  MIRROR_NODE_RETRIES: {{ .Values.config.MIRROR_NODE_RETRIES | quote }}
-  MIRROR_NODE_RETRY_DELAY: {{ .Values.config.MIRROR_NODE_RETRY_DELAY | quote }}
-  MIRROR_NODE_URL: {{ .Values.config.MIRROR_NODE_URL | quote }}
-  SUBSCRIPTIONS_ENABLED: "true"
```

Comment:
Since this was hardcoded, should we not add it to the .Values.config?

Reply:
https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/charts/hedera-json-rpc-relay-websocket/values.yaml#L23
This is currently defaulted and hardcoded to `true`.  This change is just pulling in that whole block of `.Values.config` instead of mapping them one-by-one

Reply:
makes sense. 👍